src/api/app/assets/stylesheets/webui/requests.scss

```diff
@@ -1,14 +1,14 @@
-.badge-added {
+.bg-added {
```

Comment:
For the cases of `.badge-added`, `.badge-deleted` and `.badge-changed`, kind of the same as pointed out before:
- They don't only modify the background.
- They are only applied to badges.
- They could get confused with the default .bg- Bootstrap 5 classes.